python/helpers/pydev/_pydev_bundle/_pydev_calltip_util.py

```diff
@@ -0,0 +1,160 @@
+'''
+Id:          "$Id$"
+Copyright:   Copyright (c) 2015 Bank of America Merrill Lynch, All Rights Reserved
```

Comment:
Please change the header. I had my IDE at home accidentally set to insert
this.

On Tuesday, August 30, 2016, Dmitry Trofimov <notifications@github.com>
wrote:

> In python/helpers/pydev/_pydev_bundle/_pydev_calltip_util.py
> <https://github.com/JetBrains/intellij-community/pull/439#discussion_r76839876>
> :
>
> > @@ -0,0 +1,160 @@
> > +'''
> > +Id:          "$Id$"
> > +Copyright:   Copyright (c) 2015 Bank of America Merrill Lynch, All Rights Reserved
>
> Actually our CLA doesn't transfer the copyright it only grants the
> license, but it was you, Yuli Fitterman, who signed the CLA, not Bank of
> America, so we assume that you are the copyright owner in this case.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JetBrains/intellij-community/pull/439/files/4484907fe92f3372a6c9bfa28307a9b50c8e021f#r76839876>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJZNNaLlOaGGdBwWw9iX0_xxQ5h_ClP_ks5qlGbsgaJpZM4JvEQN>
> .
>